index.html

```diff
@@ -159,19 +159,23 @@
       <div id="general-panel">
         <label class="setting-switch" for="auto-archive">
           <input type="checkbox" id="auto-archive">
-          <span class="toggle">Auto Save Page.</span>
+          <div class="setting-text-box">
+          <span class="toggle">Auto Save Page.</span></div>
```

Comment:
Please take care of indentation. Check the closing tag of div

Reply:
Apologies @anishsarangi